security/sample-template/blueprint.yaml

```diff
@@ -0,0 +1,142 @@
+apiVersion: xl/v1
+kind: Blueprint
+metadata:
+  projectName: Security and Compliance Demo Application
+  description: |
+    The blueprint creates a simple template to orchestrate Security and Compliance with XL Release.
+  author: XebiaLabs
+  version: 1.0
+spec:
+  parameters:
+  - name: AppName
+    type: Input
+    description: What is the Application name?
+  - name: ConfigureSonar
+    type: Confirm
+    description: Do you want to configure a Sonarqube compliance?
```

Comment:
I'm wondering if the question is the right way to ask it "Do you want to configure a Sonarqube compliance?"
Probably I would say something like "Do you want to embed Sonarqube in you pipeline?"

@amyjohnston probably has a good English suggestion or @nudalova a DX style?

Reply:
i'm rephrasing the question and merging. We can polish further with @amyjohnston later if needed

Reply:
I have rephrased and changed wording of all the questions